t/sanity.t

```diff
@@ -1372,3 +1372,204 @@ healthcheck: peer \[0:0::1\]:12356 was checked to be ok
 ){3,7}$/
 --- timeout: 6
 --- skip_eval: 8: system("ping6 -c 1 ::1 >/dev/null 2>&1") ne 0
+
+
+
+
+
+=== TEST 15: hook_down fires as expected
```

Comment:
"fires when peer is marked as down" might give the reader some more context as to when the function is expected to be called?